charts/prometheus-pushgateway/values.yaml

```diff
@@ -6,6 +6,9 @@ image:
   tag: v1.2.0
   pullPolicy: IfNotPresent
 
+# Names of imagePullSecrets to use.
+imagePullSecretsNames: []
```

Comment:
Why not "image.pullSecretNames"?

Reply:
Just reflecting Deployment spec structure where `imagePullSecrets` and `image` are unrelated. No strong opinion though, let me know if you want this changed.

Reply:
Let's see if there is somebody else with an opinion :)

Reply:
Looks there's no-one with an opinion around, shall we toss a coin?

Reply:
https://www.youtube.com/watch?v=hqbS7O9qIXE&t=55s :D

Reply:
I guess it's better to use imagePullSecrets instead of both your variants.
Because that's the name which used in the most of charts.

Reply:
Changed as requested.
It's a shame though as `imagePullSecrets` is already used in k8s pod spec and it takes an array of `name: <SECRET_NAME>`-s. That sort of suggests incorrect configuration in helm charts (one would try to chuck `name:` in). Not a big deal though, consistency first.